smartsim/_core/_cli/__main__.py

```diff
@@ -32,7 +32,11 @@
 def main() -> int:
     smart_cli = default_cli()
 
-    return smart_cli.execute(sys.argv)
+    try:
+        return smart_cli.execute(sys.argv)
+    except KeyboardInterrupt:
+        print("Dashboard terminated by user")
```

Comment:
Could we still dump a stack trace here if a user as their log level set to `DEBUG` or `DEVELOPER`? It seems like it could be nice it someone is making a new target, accidentally creates an infinite loop, and wants to inspect the stack to see exactly _where_ the loop was occuring

Reply:
Check out the addition of normal logging .... I think this might meet our goal